verification/io/dataplane_abstract.gobra

```diff
@@ -77,6 +77,7 @@ pure func (dp DataPlaneSpec) GetNeighborIA(ifs IO_ifs) IO_as {
 	return dp.neighborIAs[ifs]
 }
 
+// This function corresponds to function 'asid' in the IO spec
 ghost
 decreases
 pure func (dp DataPlaneSpec) GetLocalIA() IO_as {
```

Comment:
I would call it `Asid()` instead of `GetLocalIA()`. I think that using a shorter name will make the specs more readable. Same for `Core()`